partner_industry_parent/views/res_partner_industry.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright 2019 ACSONE SA/NV
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+    <!-- Inherit Form View to Modify it -->
+    <record id="res_partner_industry_view_form" model="ir.ui.view">
+        <field name="name">Industry (in partner_industry_parent)</field>
+        <field name="model">res.partner.industry</field>
+        <field name="inherit_id" ref="base.res_partner_industry_view_form"/>
+        <field name="arch" type="xml">
+
+            <xpath expr="//field[@name='name']" position="after">
```

Comment:
@sbejaoui use a direct field, remove xpath.

Reply:
Thank you @nikul-serpentcs for reviewing this.

Why would you use direct field instead of xpath?

Reply:
direct field is more easy to read, although in this case the difference is minimal